src/components/Tags/docs/index.story.tsx

```diff
@@ -169,8 +169,29 @@ export default {
                     defaultColor: 'color-8',
                   },
                 ],
-                fonts: [],
+                fonts: [
+                  {
+                    label: 'Font',
+                    wixParam: 'textFont',
+                    defaultFont: 'arial',
+                    size: 16,
+                  },
+                ],
                 numbers: [
+                  {
+                    label: 'Vertical Spacing',
+                    wixParam: 'verticalSpacing',
+                    defaultNumber: 4,
+                    unit: 'px',
+                    max: 30,
+                  },
+                  {
+                    label: 'Horizontal Spacing',
+                    wixParam: 'horizontalSpacing',
+                    defaultNumber: 4,
+                    unit: 'px',
+                    max: 30,
+                  },
```

Comment:
```suggestion
                  {
                    label: 'Vertical Spacing',
                    wixParam: 'verticalSpacing',
                    defaultNumber: 4,
                    unit: 'px',
                    max: 20,
                  },
                  {
                    label: 'Horizontal Spacing',
                    wixParam: 'horizontalSpacing',
                    defaultNumber: 4,
                    unit: 'px',
                    max: 20,
                  },
```
Harel asked to limit the max to `20px`